README.md

```diff
@@ -15,7 +21,8 @@ name: ci
 on:
   pull_request:
     paths:
-      - 'metadata/**.yaml'
+      - 'metadata/databases/**.yaml'
+      - 'metadata/version.yaml'
```

Comment:
This wouldn't work with `config v2`, but perhaps most people are off that by now?

It also wouldn't inherently pick up new features (i.e. if we were to start parsing more metadata), but I'm not sure that's going to happen anytime soon.

Reply:
> it also breaks the build because [sticky-pull-request-comment](https://github.com/marocchino/sticky-pull-request-comment) receives an empty message output from this action

Ah yeah, so we actually conditionally run the comment step in our context:

```yaml
if: steps.hasura-change.outputs.change_html
```

That might be the better README update per my comment above? Either way, it's an issue and good catch.